gateway/sds_gateway/api_methods/helpers/reconstruct_file_tree.py

```diff
@@ -12,6 +12,19 @@
 from sds_gateway.users.models import User
 
 
+def is_metadata_file(file_name: str, capture_type: CaptureType) -> bool:
+    if capture_type == CaptureType.RadioHound:
+        return file_name.endswith(".rh.json")
+
+    if capture_type == CaptureType.DigitalRF:
+        # DigitalRF metadata files contain "properties" in the name
+        # drf_properties.h5 and dmd_properties.h5 are metadata files
+        return "properties" in file_name
+
```

Comment:
Let's log an error if the capture type is not recognized (else); it'll help see a lacking implementation when we add SigMF.